### ISSUE TITLE
Documentation for installing Swift 2.0 via Cocoapods don't work

### ISSUE THREAD

**Issue by allending** [realm/realm-cocoa#2210]
- The current install instructions for Realm for Swift 2.0 via Cocoapods in the documentation does not work (https://realm.io/docs/swift/latest/#swift-20) for iOS projects.
- Pointing this out in case on one else is aware of it.
- Xcode 7b3, Cocoapods 0.37.2, Realm 0.93.2

Steps to reproduce

- Create a plain vanilla master-detail iOS project from the Xcode project template. Omit the unit test targets to keep things simple.
- Quit Xcode.
- Create a Podfile, as in per the instructions at https://realm.io/docs/swift/latest/#swift-20.
- Run `pod install`.
- Open the newly created workspace file.
- First problem: build the only available scheme for a Simulator target. A linker error occurs.

```
ld: in /Users/xxxxx/xxxxx/TestProject/Pods/Realm/core/librealm-ios.a(simulated_failure-iPhoneSimulator.o), building for iOS simulator, but linking in object file built for OSX, for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
- Okay, try building it for a real device instead. The build proceeds further than before. But second problem: the realm pods are build with bitcode enabled by default but the librealm static library isn't built with bitcode.

```
librealm-ios.a(bptree-iPhoneOS.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64
```

- Okay, disable bitcode on all the Pod realm targets and the app target. Build. Things should work.
- I wasn't able to get past the first problem though.

**Comment by segiddins** [comment]
Yup, the first problem is one we're aware of and @tgoyne is working to fix. The second will be addressed with https://github.com/realm/realm-cocoa/pull/2139 shortly!

**Comment by allending** [comment]
Thanks @segiddins. Good to hear!

**Comment by segiddins** [comment]
Sorry for failing to circle back, but I believe this has been resolved with 0.94.
